### PR TITLE
fix bugs in dashboard and create/import pages, fixes #826 , fixes #846

### LINKED ISSUE #826: Account creation process, buttons
Buttons should all have cursor: pointer (ideally cursor: pointer) should be defaulted in the button classes

When creating a new account, there is no way to go back to login page after generate mnemonic

In general, verify all the buttons of the process (creation and import)

### LINKED ISSUE #846: Bug in dashboard and multisig pages
image

### PR DESCRIPTION
**Pull request by domakun**
- add cursor:pointer in create/import account pages, #826
- fix node info in dashboard, #846 
- hide expired mosaics in dashboard, #833
-  fix inverse icon in menu bar

### CODE REVIEW

**Comment by decentraliser** [comment]
missing a `@ts-ignore` on the this=that in `NumberGrowTs.ts`

File: src/views/multisig/multisig-tree/MultisigTree.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
     </div>
     <div>
       <span class="green">{{ $t('min_removal') }}:</span>
-      rentAccountMultisigInfo ? currentAccountMultisigInfo.minRemoval : 0 }}
+      {{rentAccountMultisigInfo ? currentAccountMultisigInfo.minRemoval : 0 }}
```
Comment (decentraliser) [inline src/views/multisig/multisig-tree/MultisigTree.vue]: try with currentAccountMultisigInfo maybe

Comment (domakun) [reply src/views/multisig/multisig-tree/MultisigTree.vue]: fixed by merge other pr

File: .eslintrc.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ module.exports = {
    "@typescript-eslint",
  ],
  rules: {
+   "@typescript-eslint/no-this-alias": 0,
```
Comment (decentraliser) [inline .eslintrc.js]: Please keep that rule

Comment (domakun) [reply .eslintrc.js]: fixed

**Pull request merged**